### PR TITLE
implement syntethic feature in OutreachProgramsDashboardFragment.kt

### LINKED ISSUE #244: implement syntethic feature in OutreachProgramsDashboardFragment.kt
to reduce the number of codes and increase performance

### PR DESCRIPTION
**Pull request by kulloveth**
Fixes #244

### CODE REVIEW

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/welcome/WikiEducationDashboardFragment.kt [hunk 1]
```diff
@@ -56,8 +56,8 @@ class WikiEducationDashboardFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.fragment_wiki_edu_dashboard, container, false)
-        cv_signup_wikipedia = view.findViewById(R.id.cv_signup_wikipedia)
-        cv_login_wikipedia = view.findViewById(R.id.cv_login_wikipedia)
+        cv_signup_wikipedia = view.findViewById(R.id.cvSignUpWikipedia)
+        cv_login_wikipedia = view.findViewById(R.id.cvLoginWikipedia)
```
Comment (ujjwalagrawal17) [inline app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/welcome/WikiEducationDashboardFragment.kt]: not needed.

Comment (kulloveth) [reply app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/welcome/WikiEducationDashboardFragment.kt]: this PR is for outreachProgramsDashboardFragment.kt

**Comment by ujjwalagrawal17** [comment]
@kulloveth can you resolve merge conflicts ?

**Comment by kulloveth** [comment]
> @kulloveth can you resolve merge conflicts ?

ok, on it

**Comment by kulloveth** [comment]
> > @kulloveth can you resolve merge conflicts ?
> 
> ok, on it

done

**Comment by ujjwalagrawal17** [comment]
Thanks @kulloveth

**Pull request merged**